CRM/Core/BAO/Cache.php

```diff
@@ -75,7 +75,7 @@ public static function &getItem($group, $path, $componentID = NULL) {
       $cache = CRM_Utils_Cache::singleton();
       $cleanKey = self::cleanKey($argString);
       self::$_cache[$argString] = $cache->get($cleanKey);
-      if (!self::$_cache[$argString]) {
+      if (self::$_cache[$argString] === NULL) {
```

Comment:
Aside: on skimming, I had a knee-jerk reaction of using `isset()`/`array_key_exists()` and about handling of literal `NULL` values... but the line as written does make sense. 

Firstly, we are guaranteed to have some value in this line (b/c of the preceding `$cache->get()`).

Secondly, there is arguably an issue when the value is a purposeful `NULL` (i.e. it still does the extra lookup), but that's a rare scenario, and the cure would likely be as bad as the disease. How so? To disambiguate a purposeful `NULL` from a cache-miss-`NULL`, you either need to change the contract for `$cache` (veering away from PSR-16) or do an extra lookup (`$cache->has(...)`). The former is unpleasant from a qualitative/architectural standpoint. The latter would add an extra Redis lookup for *all cache-misses* (**somewhat common**) in order to avoid a SQL lookup for `NULL` values (**uncommon**). Evaluating that micro-optimization turns into numbers game that I don't really want to play. It's easier to KISS.

Generally, I think this PR is doing good if it just improves the cache performance for other FALSE-ish values (`0`, `'0'`, `FALSE`, `array()`). Don't sweat `NULL`.